src/Aws/EventBridgeClient.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Common\Aws;
+
+use Aws\EventBridge\EventBridgeClient as AWSEventBridgeClient;
+
+/**
+ * Class EventBridgeClient
+ * @package Common\Aws
+ */
+class EventBridgeClient extends AWSEventBridgeClient
+{
+    public function publishEvent(string $type, array $data, array $entryParams = [])
+    {
+        $meta = [];
+        if (class_exists('DDTrace\\GlobalTracer')) {
+            $meta = [
+                'meta' => [
+                    'DD_TRACE_ID' => \DDTrace\trace_id(),
+                    'DD_SPAN_ID' => \dd_trace_peek_span_id(),
+                ]
```

Comment:
Should this be merged with `$data['meta']` ?

Reply:
I think it would be nice to leave the door open for adding custom stuff to `meta`, so yes, would be nice (at least I did it for the node implementation)